Cargo.toml

```diff
@@ -16,11 +16,21 @@ exclude = ["benches/**"]
 bitflags = "1.2"
 bytemuck = { version = "1.5", features = ["extern_crate_alloc"] }
 smallvec = "1.6"
-ttf-parser = "0.15"
+ttf-parser = { version = "0.15", default-features = false, features = [
```

Comment:
I don't think this change is necessary.

Reply:
`std` is enabled by default on ttf-parser 0.15:
```toml
default = ["std", "opentype-layout", "apple-layout", "variable-fonts", "glyph-names"]
```